fluent-package/templates/etc/init.d/td-agent.erb

```diff
@@ -1,32 +1,32 @@
 #!/bin/sh
 ### BEGIN INIT INFO
-# Provides:          <%= project_name %>
+# Provides:          <%= package_dir %>
```

Comment:
Hmm, some variables that shouldn't be `pacakge_dir` are also modified...

Reply:
In this PR, it keeps the path as same as before, so it seems that `Provides: td-agent` is reasonable.
Am I missing context here?

Reply:
I'm talking about the variable name. I mean the variable name should express a name of a package, it's not a directory. There may be no suitable variable yet though.

Reply:
Should we introduce a new variable like `service_name`?

Reply:
:+1:  Yes, I was thinking about introducing that name to express them.

Reply:
fixed.